### PR TITLE
Don't skip stroke path

### PR DESCRIPTION
**Pull request by sangwoo108**
Even when it doesn't fill, we should check it's stroke path.

---

Hello! I think we should generate a path when it's a stroke. PTAL and thank you in advance.

### CODE REVIEW

**Comment by evanstade** [comment]
Thanks for the patch. Do you have an example of an SVG for which this fixes skiafy's output?

**Comment by sangwoo108** [comment]
Here it is. :)

```svg
<svg width="16" height="16" xmlns="http://www.w3.org/2000/svg">
  <path d="M11.98 15.7L4 8 11.98.3" stroke="#202429" fill="none" fill-rule="evenodd"/>
</svg>
```

**Comment by evanstade** [comment]
Thanks. I think we also need to insert a STROKE command in that case then? Shouldn't be too hard to add I hope. Otherwise, with your patch, we'll get a path in the .icon file which fills instead of stroking. This is not much better than being absent instead of stroking. (Neither case gives very easily debuggable output.)

File: main.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ function ToCommand(letter) {
     case 'z':
       return 'CLOSE';
   }
-  return '~UNKNOWN~';
+  return letter;
```
Comment (sangwoo108) [inline main.js]: What do you think about this line? As the stroke is not a command from the 'd', I just changed like this.
Maybe we can either do `case 'STROKE' return 'STROKE'`.

File: main.js [hunk 2]
```diff
@@ -82,12 +82,22 @@ function HandleNode(svgNode, scaleX, scaleY, translateX, translateY) {
...
           break;
 
         var commands = [];
+        if (isStrokePath) {
+          var strokeWidth =  svgElement.getAttribute('stroke-width');
+          if (!strokeWidth || isNan(strokeWidth))
+            strokeWidth = 1;
+
+          commands.push({ 'command': 'STROKE', 'args': [strokeWidth] });
+        }
```
Comment (evanstade) [inline main.js]: can we just avoid using |commands|, and append to |output| directly? Just like in the non-'path' cases.

Comment (sangwoo108) [reply main.js]: That's nice! Done.

**Comment by evanstade** [comment]
thanks for the change!

**Comment by sangwoo108** [comment]
My pleasure. Thank you for the review. See you around!

**Pull request merged**